src/components/molecules/UserProfileHeader.tsx

```diff
@@ -19,15 +21,20 @@ const useStyles = makeStyles((theme) => ({
   welcomeMsg: {
     padding: theme.spacing(1),
   },
+  avatar: {
+    width: theme.spacing(5),
```

Comment:
do not use `theme.spacing` for height / width
See `fontSize`
https://material-ui.com/api/icon/

Reply:
I'm using  `Avatar` component,  see https://material-ui.com/components/avatars/#sizes

Reply:
still,  using spacing for size seems like a bad practice (although appears in the docs)